drivers/interrupt_controller/Kconfig.vim

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if CPU_CORTEX
+
+config VIM
+	bool
```

Comment:
Please, check how this is done for the other interrupt controllers. You are missing description, dependencies, etc...